hub/hub.go

```diff
@@ -59,10 +59,8 @@ const (
 
 // Message types
 const (
-	//transform = 0
-	//rpc = 1
-	openMessage  = 2
-	closeMessage = 3
+	openMessage  = iota
+	closeMessage = iota
```

Comment:
In this case, I think that there is only one iota, but do you have any reason?

https://play.golang.org/p/w9KX9ry0Da
https://play.golang.org/p/Z42CVBe3ZbC

Reply:
I read only the first example of this article. I don't have a reason.
https://qiita.com/curepine/items/2ae2f6504f0d28016411